packages/trader/src/Stores/Modules/Portfolio/portfolio-store.js

```diff
@@ -81,7 +84,7 @@ export default class PortfolioStore extends BaseStore {
             if (i === -1) return;
 
             this.positions[i].is_loading = true;
-            WS.subscribeProposalOpenContract(contract_id, this.populateResultDetails, false);
+            WS.send({ proposal_open_contract: 1, contract_id }).then(this.populateResultDetails);
```

Comment:
how come we don't use a method here?

Reply:
Yes we should. At first we where going to use caching in the methods and use `send` call to make sure caches are not used. But I think it's a better idea to let the user decide to use the caching or not, by specifying `WS.cache` or `WS.storage` anywhere needed.

We currently have only one of such case: https://github.com/binary-com/deriv-app/pull/284/files#diff-733e4e695af65300ab7b8677a7c765daR171 which should be easy enough to fix.